tests/chainer_tests/functions_tests/connection_tests/test_deconvolution_2d.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class TestDeconvolution2D(unittest.TestCase):
```

Comment:
I would like you to make this test case parametrized with respect to input size, kernel size, stride and pad size, since one of the most difficult parts of (deconvolution) is consistency of input size and output size in edge cases.